kernel/sys/ports.h

```diff
@@ -0,0 +1,55 @@
+//! @file ports.h
+//! @brief File containing functions for interfacing with IO ports on x86
```

Comment:
"File containing functions for interfacing with IO ports on x86 architecture" is probably more contrived than it should have been
"x86 IO port functions" is a better name in my opinion, as "File containing functions for interfacing" is just crust

Reply:
Fixed